doc/version_history.rst

```diff
@@ -6,9 +6,26 @@
 Version History
 ###############
 
+v0.1.1
+------
+
+Changes:
+
+* Add a Jenkinsfile
```

Comment:
Maybe add a "... for CI job." because we usually have a Jenkinsfile for conda as well..